packages/ibm-products/src/components/Datagrid/Datagrid/DatagridContent.js

```diff
@@ -252,6 +252,7 @@ DatagridContent.propTypes = {
       PropTypes.func,
     ]),
     isFetching: PropTypes.bool,
+    skeletonRows: PropTypes.number,
```

Comment:
This is good, but think more consistent naming with Carbon might be better if we’re introducing a new prop. The DataTable uses `rowCount`, but think we should still specify skeleton in this case.

```suggestion
    skeletonRowCount: PropTypes.number,
```